packages/@okta/i18n/src/properties/login.properties

```diff
@@ -33,6 +33,10 @@ sensitive.input.show = Show
 sensitive.input.hide = Hide
 processing.alt.text = Processing...
 logo.default.alt.text = Logo
+icon.title.application = Application
+icon.title.browser = Browser
+icon.title.location = Location
+icon.title.mobileDevice = Mobile Device
```

Comment:
You may also want to create a PR into master too for these so they are translated and hits the next monthly translation request.